### ISSUE TITLE
Deprecate some arguments in `start_scan` and refactor fallback seed

### ISSUE THREAD

**Issue by mlincett** [icecube/skymap_scanner#230]
The following arguments of `start_scan`:
1. `input_time_name: str`
2. `input_pos_name: str`
3. `output_particle_name: str`
define the starting seed (`fallback_*`) and the current seed `output_particle_name`.

These arguments are never used and they always correspond to default values.

What reco modules do now is write the starting seed to the default value of `input_(pos|time)_name` (server-side) which is `cfg.INPUT_(POS|TIME)_NAME` and read the current seed from the default value of  `output_particle_name` (client-side) which is `cfg.OUTPUT_PARTICLE_NAME`.

In other words, there is no benefit in defining any of this in `start_scan`. The reco module could provide a name for the starting seed (optionally, more than one) while `output_particle_name` is purely conventional.

**Comment by tianluyuan** [comment]
@mlincett are you able to get to this still?

**Comment by mlincett** [comment]
> @mlincett are you able to get to this still?

I'd like to, but I have some other things on my backlog still.
